packages/mui-lab/src/Masonry/Masonry.test.js

```diff
@@ -25,89 +24,128 @@ describe('<Masonry />', () => {
   );
 
   const theme = createTheme({ spacing: 8 });
-
+  const maxColumnHeight = 100;
+  const maxNumOfRows = 5;
+  const parseToNumber = (val) => {
+    return Number(val.replace('px', ''));
+  };
   describe('style attribute:', () => {
```

Comment:
It would be great if you could test (in addition to or instead) the actual rendered Masonry component, not just the `style`.